src/openrct2/ride/VehicleSubpositionData.cpp

```diff
@@ -21932,80 +24395,116 @@ static constexpr const rct_vehicle_info_list *TrackVehicleInfoList_8BD228[] = {
+    &LeftLane::TrackVehicleInfoLeftSBend0, &LeftLane::TrackVehicleInfoLeftSBend1, &LeftLane::TrackVehicleInfoLeftSBend2, &LeftLane::TrackVehicleInfoLeftSBend3, // 38 s-bend left *
+    &LeftLane::TrackVehicleInfoRightSBend0, &LeftLane::TrackVehicleInfoRightSBend1, &LeftLane::TrackVehicleInfoRightSBend2, &LeftLane::TrackVehicleInfoRightSBend3, // 39 s-bend right *
+    &TrackVehicleInfo_8D44BE, &TrackVehicleInfo_8D4FA6, &TrackVehicleInfo_8D5A8E, &TrackVehicleInfo_8D6576, // 40 left loop
+    &TrackVehicleInfo_8D705E, &TrackVehicleInfo_8D7B46, &TrackVehicleInfo_8D862E, &TrackVehicleInfo_8D9116, // 41 right loop
+    &LeftLane::TrackVehicleInfoLeftQuarterTurn3Tiles0, &LeftLane::TrackVehicleInfoLeftQuarterTurn3Tiles1, &LeftLane::TrackVehicleInfoLeftQuarterTurn3Tiles2, &LeftLane::TrackVehicleInfoLeftQuarterTurn3Tiles3, // 42 small left
+    &LeftLane::TrackVehicleInfoRightQuarterTurn3Tiles0, &LeftLane::TrackVehicleInfoRightQuarterTurn3Tiles1, &LeftLane::TrackVehicleInfoRightQuarterTurn3Tiles2, &LeftLane::TrackVehicleInfoRightQuarterTurn3Tiles3, // 43 small right
+    &LeftLane::TrackVehicleInfoLeftQuarterTurn3Tiles0, &LeftLane::TrackVehicleInfoLeftQuarterTurn3Tiles1, &LeftLane::TrackVehicleInfoLeftQuarterTurn3Tiles2, &LeftLane::TrackVehicleInfoLeftQuarterTurn3Tiles3, // 44 banked small left
+    &LeftLane::TrackVehicleInfoRightQuarterTurn3Tiles0, &LeftLane::TrackVehicleInfoRightQuarterTurn3Tiles1, &LeftLane::TrackVehicleInfoRightQuarterTurn3Tiles2, &LeftLane::TrackVehicleInfoRightQuarterTurn3Tiles3, // 45 banked small right
+    &LeftLane::TrackVehicleInfoLeftQuarterTurn3Tiles25DegUp0, &LeftLane::TrackVehicleInfoLeftQuarterTurn3Tiles25DegUp1, &LeftLane::TrackVehicleInfoLeftQuarterTurn3Tiles25DegUp2, &LeftLane::TrackVehicleInfoLeftQuarterTurn3Tiles25DegUp3, // 46 up small left
+    &LeftLane::TrackVehicleInfoRightQuarterTurn3Tiles25DegUp0, &LeftLane::TrackVehicleInfoRightQuarterTurn3Tiles25DegUp1, &LeftLane::TrackVehicleInfoRightQuarterTurn3Tiles25DegUp2, &LeftLane::TrackVehicleInfoRightQuarterTurn3Tiles25DegUp3, // 47 up small right
+    &LeftLane::TrackVehicleInfoLeftQuarterTurn3Tiles25DegDown0, &LeftLane::TrackVehicleInfoLeftQuarterTurn3Tiles25DegDown1, &LeftLane::TrackVehicleInfoLeftQuarterTurn3Tiles25DegDown2, &LeftLane::TrackVehicleInfoLeftQuarterTurn3Tiles25DegDown3, // 48 down small left
+    &LeftLane::TrackVehicleInfoRightQuarterTurn3Tiles25DegDown0, &LeftLane::TrackVehicleInfoRightQuarterTurn3Tiles25DegDown1, &LeftLane::TrackVehicleInfoRightQuarterTurn3Tiles25DegDown2, &LeftLane::TrackVehicleInfoRightQuarterTurn3Tiles25DegDown3, // 49 down small right
+    &TrackVehicleInfo_901C05, &TrackVehicleInfo_901C7C, &TrackVehicleInfo_901CFC, &TrackVehicleInfo_901D73, // 50 tiny left
+    &TrackVehicleInfo_901762, &TrackVehicleInfo_901884, &TrackVehicleInfo_9019AF, &TrackVehicleInfo_901AE3, // 51 tiny right
 };
 
+// go karts right lane
```

Comment:
Rather than putting this in a comment (which _is_ an improvement upon the current situation), could you rename the variable to `TrackVehicleInfoListGoKartsRightLane`?